ui/src/containers/NodePageRSP.js

```diff
@@ -121,13 +122,10 @@ const NodePageRSP = (props) => {
 
   // Filter alerts for the specific node, base on the InstaceIP and Alert Name
   const alerts = useSelector((state) => state.app.alerts.list);
-  const alertsNode =
-    alerts?.filter(
-      (alert) =>
-        NODE_ALERTS_GROUP.includes(alert?.labels?.alertname) &&
-        `${instanceIP}:${PORT_NODE_EXPORTER}` === alert?.labels?.instance,
-    ) ?? [];
-
+  const alertsNode = filterAlerts(alerts, {
```

Comment:
I think alerts should be dropped from redux store to avoid duplicating the source of alerts for the application. So instead of calling filterAlerts here we should use useAlerts hook